bc_obps/Makefile

```diff
@@ -92,6 +92,11 @@ superuser: ## create a superuser
 superuser:
 	$(POETRY_RUN) python $(MANAGE_PY) create_superuser
 
+loadfixtures_v1: ## add all registration fixture files to the db
```

Comment:
I'm a bit confused by the naming conventions - is `loadfixtures_v1` for Reg1 and `loadfixtures` for Reg2?

Reply:
Yes, I'll make a comment. This is to match how we've done the nx apps and the endpoints (so when we delete reg1 we're just left with reg instead of reg2, v2, etc.)